stable/openldap/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- enis
```

Comment:
While this is certainly not a blocker, we can remove this or assign appropriate approvers/maintainers

Reply:
I'm not a maintainer of this chart. Don't add me.

Reply:
updated